### ISSUE TITLE
Hello and thank you for your wonderful work. I would like to ask again if you have done some preprocessing on the scale of the depth map inferred by the depth estimation model, because I found that it is often difficult to directly use the depth map obtained by depth estimation for warp to align the target perspective pose and depth value. The scale size greatly affects the warp operation.

### ISSUE THREAD

**Issue by chenkangjie1123** [eckertzhang/Text2NeRF#12]
Hello and thank you for your wonderful work. I would like to ask again if you have done some preprocessing on the scale of the depth map inferred by the depth estimation model, because I found that it is often difficult to directly use the depth map obtained by depth estimation for warp to align the target perspective pose and depth value. The scale size greatly affects the warp operation.

**Comment by eckertzhang** [comment]
yes, we need to pre-process the depth map estimated by the depth estimation model, please refer the line 200 of [dataLoader](https://github.com/eckertzhang/Text2NeRF/blob/73c2745744fb26276aa7293fd9b7c4e9933977ae/dataLoader/scene_gen.py#L200) for more details.
